specification/quota/resource-manager/Microsoft.Quota/preview/2021-03-15-preview/quota.json

```diff
@@ -0,0 +1,1226 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-03-15-preview",
+    "title": "Azure Quota Extension API",
+    "description": "Microsoft Azure Quota Resource Provider."
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{resourceUrl}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/resourceUrl"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
```

Comment:
>"paths" [](http://example.com/codeflow?start=2&length=7)

Just making sure, these are tenant level apis right?

Reply:
It can be at subscription level or Tenant level APIs, as it depends on the RPs, at what level of granularity, they need.
For example - Batch allocates quota at - /subscription/location/batchAccounts/{batchAccountname}, CosmosDb and SQL 
at global level, no location. 

This parameter will determine the resource context based on ARM extension model -
"x-ms-parameterized-host": {
    "hostTemplate": "{resourceUrl}",
    "useSchemePrefix": false,
    "parameters": [
      {
        "$ref": "#/parameters/resourceUrl"
      }

Reply:
Does this actually work with the generated SDKs?

Reply:
Updated to {scope} as suggested.